client/src/domains/constants.ts

```diff
@@ -7,6 +7,18 @@ export const MOBILE_MAX_WIDTH = 575.98;
 
 export const PAGE_CNT = 5;
 
+export const MENU_HEIGHT = 50;
+export const TUTORIAL_CONTENTS = {
+  EDITOR: "1. 에디터고 여기서 편집을 할 수 있으며 기본으로 제공되는 양식은 발표의 첫 페이지이며 없애면 이상해져요.",
+  PREVIEW: "2. 마크다운 적용 결과고요 미리 볼 수 있어요.",
+  MENU: "3. 각종 기능들인데요, 기존 마크다운 문법과 동일해요. 근데 사진은 드래그 앤 드롭이 가능하구요! 유튭이랑 차트는 설명이 좀 필요할 것 같아요.",
```

Comment:
Try out example buttons at the top to check supported features.
You can use all of the GFM style syntax, and additional support such as Youtube and Chart are currently available. 
We will support more features in the future, so stay tuned!